core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAOSpec.scala

```diff
@@ -137,15 +137,24 @@ class HttpGoogleServicesDAOSpec extends AnyFlatSpec with Matchers with MockitoTe
 
   behavior of "setupWorkspace"
 
-  it should "create the workspace's GCS bucket" in {
+  /*
+    FIXME: This test has likely been broken for its entire existence, so marking it as ignored for now
+       the test was not waiting for the future returned by the call to googleServicesDAO.setupWorkspace to complete before moving on.
+       So it was failing (when verifying the call to googleStorageService.insertBucket),
+       but the test framework would wait for child processes to complete before reporting.
+       If another test was executing when it was reported, the failure would show up in that test,
+       but with the actual failure point listed as this test, far down in the stack trace (or however deep in the call stack it was when it failed)
+       If no test was active, whether because all tests had completed, or the framework was in between tests, the failure would not be reported at all.
+   */
+  it should "create the workspace's GCS bucket" ignore {
```

Comment:
I could also replicate the behavior on develop, by running all tests in the `dataaccess` package.

Reply:
👍 This should get a tech debt ticket.